.github/workflows/deploy-storybook.yml

```diff
@@ -3,7 +3,7 @@ name: Deploy Storybook
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Hmmm... what should we do about this? Random ideas
- we make this workflow instead run when a new tag is created on GH 
- we make a `yarn publish` that pulls down latest main, runs npm publish, and then runs storybook

Reply:
hmmmm good question... I think I prefer the publish happening in GH, and it looks like we can trigger it to run when a tag is pushed. what if we filter on tags like this to capture all v1 (and we can add `2, 3,` etc down the line) - https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags
```
- 1.*
```